### DISCUSSION TITLE
clicking on an asset icon or name should always lead to the relevant asset page

### DISCUSSION THREAD

**Discussion by aspiers** [rotki/rotki#8991] [Ideas]
When I go to `Accounts & Balances / Blockchain Balances`, and click on an asset name or icon, it navigates to the relevant asset page.  This is great!

However, if in the same table I expand one of the assets, e.g. ETH, and click on any of the asset icons in the expanded table (i.e. ETH or WETH icons), nothing happens:

![image](https://github.com/user-attachments/assets/3694b24c-c961-42cf-867c-78f3e95ae61a)

It would be much more useful if it navigated to the relevant asset page, just like it does in the main table.

The same applies for the EVM accounts table, and probably other views too:

![image](https://github.com/user-attachments/assets/80ef8195-3f08-47fc-a9d4-5ac89a113e3b)

(BTW see also https://github.com/rotki/rotki/issues/8990 for an issue with tooltips in this view.)

### System Description

Operating system: openSUSE Tumbleweed Linux
Rotki version: 1.36.1 AppImage

**Comment by LefterisJP** [comment]
That should be easy to do I guess.

**Comment by lukicenturi** [comment]
Closed by https://github.com/rotki/rotki/pull/8998

**Reply by aspiers** [reply]
Nice work @cleanerzkp and @lukicenturi!  Look forward to trying this out :rocket:
